src/main/resources/swagger.api/schemas/userTenant.yaml

```diff
@@ -8,7 +8,7 @@ UserTenant:
     userId:
       type: string
       format: uuid
-    username:
+    userName:
```

Comment:
This change can broke exsiting contract, UI for adding affiliation uses username

Reply:
Basically we need to introduce additional PR in mod-users:
User dto:
![image](https://github.com/folio-org/mod-consortia/assets/25097693/1f6c51c5-5e3f-4179-bd96-3070b4e1f5b6)
UserTenant dto username is not consistent, we need to rename it in mod-users:
![image](https://github.com/folio-org/mod-consortia/assets/25097693/f5cf4b1b-9fc7-40d1-9e49-84c6c0d5f828)

Reply:
fixed